Source/common/SNTFileAccessEvent.h

```diff
@@ -37,17 +38,4 @@
 ///
 @property NSString *application;
 
-///
-/// A string representing the publisher based on the signingChain
-///
-@property(readonly) NSString *publisherInfo;
-
-///
-/// Return an array of the underlying SecCertificateRef's of the signingChain
-///
-/// WARNING: If the refs need to be used for a long time be careful to properly
-/// CFRetain/CFRelease the returned items.
-///
-@property(readonly) NSArray *signingChainCertRefs;
```

Comment:
Don't we need this for the process that caused the file access policy violation?

Reply:
I've moved it into the superclass